mimic/canned_responses/fastly.py

```diff
@@ -0,0 +1,278 @@
+"""
+Canned response for fastly
+"""
+import uuid
+
+
+class FastlyResponse(object):
+
+    """
+    Canned response for fastly.
+    See docs here https://docs.fastly.com/api/config
+    """
+
+    fastly_cache = {}
+
+    def get_current_customer(self):
+        """
+        Returns the current customer with response code 200.
+
+        :return: a JSON-serializable dictionary matching the format of the JSON
+                 response for fastly_client.get_current_customer()
+                 ("/current_customer") request.
+        """
+        current_customer = {
+            u'can_edit_matches': u'0',
+            u'can_read_public_ip_list': u'0',
+            u'can_upload_vcl': u'1',
+            u'updated_at': u'2014-11-03T23:37:44+00:00',
+            u'has_config_panel': u'1',
+            u'has_improved_ssl_config': False,
+            u'id': u'3bKRHISECARTtoUMxW6',
+            u'has_historical_stats': u'1',
+            u'has_openstack_logging': u'0',
+            u'can_configure_wordpress': u'0',
+            u'has_improved_logging': u'1',
+            u'readonly': '',
+            u'ip_whitelist': u'0.0.0.0/0',
+            u'owner_id': u'asewQWEwdRnLnCKyMztZqkQvy',
+            u'phone_number': u'770-123-1749',
+            u'postal_address': None,
+            u'billing_ref': None,
+            u'can_reset_passwords': True,
+            u'has_improved_security': u'1',
+            u'stripe_account': None,
+            u'name': u'Poppy - Test',
+            u'created_at': u'2014-11-03T23:37:43+00:00',
+            u'can_stream_syslog': u'1',
+            u'pricing_plan': u'developer',
+            u'billing_contact_id': None,
+            u'has_streaming': u'1'}
+        return current_customer
+
+    def create_service(self, url_data):
+        """
+        Returns POST service with response json.
+
+        :return: a JSON-serializable dictionary matching the format of the JSON
+                 response for fastly_client.create_service()
+                 ("/service") request.
+        """
+        data = dict((key, value[0]) for key, value in url_data)
+
+        publish_key = uuid.uuid4().hex
+        service_id = uuid.uuid4().hex
+        service_name = data['name']
+
+        self.fastly_cache[service_name] = {
```

Comment:
This looks like a copy of the recently-removed bad pattern for Nova, where everything was a "cache" and stored in a global variable.  The thing stored here should probably evolve to be some kind of class instance, which can do JSON serialization as *one* of its tasks.

It's fine to put this off for a future revision, though.